Cargo.toml

```diff
@@ -6,6 +6,5 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-accrete = "0.2.0"
```

Comment:
Good call removing accrete, since for the time being we're not actually using it.